source/documentation/other-topics/Setup-postgres-container.md

```diff
@@ -0,0 +1,79 @@
+ - You have [Authenticated][auth-to-cluster] to the cloud-platform-live-1 cluster.
+ - You have configured [Helm and Tiller][using-helm]. 
+
+##### Set up
+First we need to clone our reference application and change directory:
+
+    $ git clone https://github.com/ministryofjustice/cloud-platform-reference-app.git
+    $ cd cloud-platform-reference-app/helm_deploy/django-app/charts/postgresql
+
+You now have a [Bitnami PostgreSQL][postgresql-chart] Helm Chart, default values for postgresql are stored in the `values.yaml`, if you need to create your own values for postgresqlUsername,postgresqlPassword and postgresqlDatabase you can provide those as an argument on our installation command, the `set postgresql values` overwrites the value stored in `value.yaml` file.
+
+Run the following (replacing the `YourName` with your own name and `env-name` with your environment name:
+
+        $ helm install . \
+          --name postgres-<YourName> \
+          --namespace <env-name> \
+          --set postgresqlUsername=postgres,postgresqlPassword=secretpassword,postgresqlDatabase=my-database \
+          --tiller-namespace <env-name>
+
+##### Viewing your PostgreSQL DB
+
+You can now check PostgreSQL Helm Chart is deployed sucessfully: 
+
+    $ kubectl get pods --namespace <env-name>
+
+If the Installation was successful you should be seeing something similar to the below:
+
+```
+NAME                              READY     STATUS    RESTARTS   AGE
+<YourName>-postgresql-0           1/1       Running   0          39m
+```
+You should have a postgres pod with the status **running**. You can also check the logs of the PostgreSQL pod: 
+
+    $ kubectl --namespace <env-name> logs <YourName>-postgresql-0 
+
+If the PostgreSQL setup was successful you should be seeing tail of log as below:
+
+```
+ 12:49:28.02 INFO  ==> ** PostgreSQL setup finished! **
+``` 
+
+##### Accessing your PostgreSQL DB
+
+PostgreSQL can be accessed via port 5432 on the following DNS name from within your cluster:
+
+    <YourName>-postgresql.<env-name>.svc.cluster.local - Read/Write connection
+
+The `postgresqlPassword` you have set will be stored as a secret in your namespace, to get the password for "postgres" run:
+
+    export POSTGRES_PASSWORD=$(kubectl get secret --namespace <env-name> <YourName>-postgresql -o jsonpath="{.data.postgresql-password}" | base64 --decode)
+
+
+To connect to your database from outside the cluster execute the following commands:
+
+    kubectl port-forward --namespace <env-name> svc/<YourName>-postgresql 5432:5432 &
+    PGPASSWORD="$POSTGRES_PASSWORD" psql --host 127.0.0.1 -U postgres -d my-database -p 5432
+
+
+Congratulations on getting this far. If all went well your postgresql pod is now deployed and you could connect to your database from outside the cluster.
+
+
+[env-create]: tasks.html#creating-a-cloud-platform-environment
+[auth-to-cluster]: tasks.html#authentication
```

Comment:
There is a mixture of absolute and relative URLs here, for links to other parts of the user guide. Please replace all the absolute URLs with relative ones.

Reply:
Thanks @digitalronin updated the URL's to relative ones.